.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -107,10 +120,27 @@ jobs:
             pytest -sv \
             --last-failed \
             --tb short \
-            --color yes
+            --color yes \
+            --splits ${{ env.PYTEST_SPLIT_GROUPS }} --group ${{ matrix.pytest-split-group }} \
+            --store-durations --splitting-algorithm least_duration --durations-path=.test_durations.${{ matrix.pytest-split-group }} --clean-durations
```

Comment:
Please put these one-per-line for easier reading.

Reply:
Will do